Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM ruby:2.3.0
+RUN apt-get update -qq && apt-get install -y build-essential libpq-dev
+RUN mkdir /work-you-could-do
+WORKDIR /work-you-could-do
+ADD Gemfile /work-you-could-do/Gemfile
+ADD Gemfile.lock /work-you-could-do/Gemfile.lock
+RUN bundle install
+ADD . /work-you-could-do
```

Comment:
Dockerfile copy-pasted from Docker compose help (https://docs.docker.com/compose/rails/) hence why it's like that. This would work without adding the Gemfile separately. I don't know if or how much of a speed difference this makes.